### ISSUE TITLE
`export default defineMessages(...)` is not detected

### ISSUE THREAD

**Issue by ccnokes** [bang88/typescript-react-intl#42]
Moving it to a variable declaration is a workaround: 

```typescript
const i18n = defineMessages(...);
export default i18n;
```
Ideally, it'd always find that `defineMessages` invocation.

**Comment by ccnokes** [comment]
Also just going to note here that `intl.formatMessage` does not seem to either.

**Comment by ccnokes** [comment]
Oops the latest version resolves this. Sorry about that!
